services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -206,10 +219,12 @@ setFeatureStatusNoConfig applyState tid status = do
     Event.Event Event.Update (Public.knownTeamFeatureName @a) (EdFeatureWithoutConfigChanged newStatus)
   pure newStatus
 
-getSSOStatusInternal :: Maybe TeamId -> Galley (Public.TeamFeatureStatus 'Public.TeamFeatureSSO)
+type GetFeatureInternalParam = Either (Maybe UserId) TeamId
```

Comment:
I guess this is alright, but for the future would it make sense to make this a type family dependent on the feature flag, so that we get more type safety?

Reply:
https://github.com/wireapp/wire-server/pull/1818/commits/5a3f4ee98cc0a12eb27936d856823d764f592dda

fully agreed, yet de-prioritized (sorry...)